### DISCUSSION TITLE
Tweeter Announcement Proposals

### DISCUSSION THREAD

**Discussion by nyurik** [maplibre/maplibre#105] [General governance]
**Update: New rule as of Jan 16th, 2023 (https://github.com/maplibre/maplibre.github.io/pull/111) is to write a news post at https://maplibre.org/news with the twitter announcement included. Merging to this website requires approval from two Governing Board members.**

Use this discussion to propose tweets that we should publish from the :bird: [@maplibre](https://twitter.com/maplibre) Tweeter account.

Start a new topic for each tweet.  Use "reply" to comment/object/correct the specific tweet. Please write clearly the exact text you think should be posted.

At least two members of the MapLibre Governing Board (currently @nyurik @lseelenbinder @HarelM @klokan @birkskyum) have to approve your tweet.

**Comment by nyurik** [comment]
Propose to post this text with the picture. Plus possibly a link to https://www.npmjs.com/package/maplibre-gl
Not sure if this is the right text, suggestions welcome

> MapLibre GL JS has just surpassed 100,000 weekly downloads on NPM!  Big thank you goes to our contributors, supporters, and users for your support and trust in what we do!  https://www.npmjs.com/package/maplibre-gl

![image](https://user-images.githubusercontent.com/1641515/190889034-098f1e25-c832-4bb1-8fd4-2e142b7c1fd0.png)


p.s.  the image for some reason gets really messed up on twitter, so some image magic might be required first

**Reply by wipfli** [reply]
We can also mirror it on LinkedIn once it is on Twitter...

**Reply by wipfli** [reply]
Since there are no strong objections, I will go ahead and publish this message on Twitter and afterwards on LinkedIn.

**Comment by wipfli** [comment]
If you are enthusiastic about maps and would like to work with a global open-source community of really nice people, MapLibre might be the opportunity for you! We are hiring. Apply now at https://maplibre.org/jobs

![Screenshot_20221011_213535](https://user-images.githubusercontent.com/53421382/195183305-efb21afb-df6f-4640-9e9f-3e3fa7fd22fa.png)

**Reply by wipfli** [reply]
https://twitter.com/maplibre/status/1580805915397103616

**Reply by wipfli** [reply]
https://www.linkedin.com/feed/update/urn:li:activity:6986573204001562624

**Comment by wipfli** [comment]
MapLibre GL Native is planning for modularization and migration to Metal, Apple's graphics framework, for rendering on iOS mobile devices. Read more...

![image](https://user-images.githubusercontent.com/53421382/198733487-c46a33e5-84b9-49da-b7e4-9950e809f231.png)

**Reply by wipfli** [reply]
MapLibre GL Native is planning for modularization and migration to Metal, Apple’s graphics framework, for rendering on iOS mobile devices.

**Reply by nyurik** [reply]
Lets put the word "planning" (or similar) in between MapLibre and Metal on iOS - this way it is clear that we are discussing it, not announcing that its ready

**Reply by wipfli** [reply]
Ok

**Reply by wipfli** [reply]
![image](https://user-images.githubusercontent.com/53421382/198865599-b455f515-3a18-40b3-8ca6-c4acc58b9d33.png)

**Reply by wipfli** [reply]
https://twitter.com/maplibre/status/1586624103393464320 and https://www.linkedin.com/feed/update/urn:li:activity:6992390722616229888

**Comment by wipfli** [comment]
Applications for jobs at MapLibre are open until Wednesday, Nov 30th, 2022.

This is your opportunity to work on an open-source maps project with an amazing community.

If you are interested, fill out the application form at https://maplibre.org/jobs/...

**Reply by wipfli** [reply]
@lseelenbinder are you good with this?

**Reply by lseelenbinder** [reply]
LGTM. Thanks @wipfli

**Reply by wipfli** [reply]
https://www.linkedin.com/feed/update/urn:li:activity:7000744655382282240

**Reply by wipfli** [reply]
https://twitter.com/maplibre/status/1594978712432398336

**Comment by wipfli** [comment]
MapLibre has a new project: **font-maker** - a web app to convert font files into fontstacks for MapLibre GL. With font-maker, you can take your favorite .ttf or .otf font files and convert them to a format usable in your MapLibre GL JS and MapLibre GL Native maps. Thanks @bdon for sharing this project with the MapLibre community!

![image](https://user-images.githubusercontent.com/53421382/205439745-2f3126b6-07ac-41d8-976d-e1e804e1b65b.png)

**Reply by HarelM** [reply]
LGTM

**Reply by nyurik** [reply]
I like it, but it might be too long for a tweet

**Reply by wipfli** [reply]
https://twitter.com/maplibre/status/1599306132828278785

**Reply by wipfli** [reply]
https://www.linkedin.com/feed/update/urn:li:activity:7005072893948624896

**Comment by wipfli** [comment]
If you have an idea for a Bounty, you can now share it at https://github.com/maplibre/maplibre/discussions/156

**Reply by nyurik** [reply]
we may want to explain what a bounty is... not certain what the wording may be

**Reply by wipfli** [reply]
Maybe:

If you have an idea for a new feature or improvement of existing code, you can now share your idea and propose a Bounty for it.

**Reply by nyurik** [reply]
Maybe focus on money?  E.g.

`Tell us how MapLibre should spend donations. Nominate which new features or bugs should be added to our bounties program.`

**Reply by wipfli** [reply]
That is too steep from my point of view.

**Reply by wipfli** [reply]
What about this?

If you have ideas for new features which you would love to see from MapLibre, you can now nominate them for the MapLibre Bounty Program. [Nominate and vote for Bounties here...](https://maplibre.org/news/2022-10-16-call-for-bounties/)

**Reply by nyurik** [reply]
Maybe a bit less verbose?  Something like

`If you have ideas for new features in MapLibre, you can now nominate them for the MapLibre Bounty Program here https://maplibre.org/news/2022-10-16-call-for-bounties/`

(Can you use Markdown links in twitter?)

**Reply by wipfli** [reply]
Beautiful

**Reply by lseelenbinder** [reply]
Looks good 👍

**Reply by wipfli** [reply]
https://twitter.com/maplibre/status/1604743855730999296

**Reply by wipfli** [reply]
https://www.linkedin.com/feed/update/urn:li:activity:7010510238902591488

**Comment by wipfli** [comment]
Today, Bart Louwers is starting to work with MapLibre in the Maintainer Native role. Let’s give him a warm welcome! [Read more...](https://maplibre.org/news/2023-01-03-announcement-bart/)

**Reply by lseelenbinder** [reply]
Looks good.

**Reply by HarelM** [reply]
Approved.

**Reply by wipfli** [reply]
https://twitter.com/maplibre/status/1610268913484406785

**Reply by wipfli** [reply]
https://www.linkedin.com/feed/update/urn:li:activity:7016035375940816897

**Comment by wipfli** [comment]
Read the latest community post on writing plugins which work for both MapLibre and Mapbox GL JS: https://maplibre.org/news/2023-01-03-aws-maplibre-plugins/

**Reply by birkskyum** [reply]
Can we show the name instead of **austyle**?

Do we have an example of a package where this is implemented already?

**Reply by birkskyum** [reply]
I can't find this intersection type snippet in the geocoder library

**Reply by wipfli** [reply]
Thanks for the feedback. Probably best to ask in the original pull request which added to news post to the website

**Reply by birkskyum** [reply]
Anyway, I think it's a great idea to make these agnostic plugins :) Approved.

**Reply by wipfli** [reply]
https://mobile.twitter.com/maplibre/status/1613075610577821702

**Reply by wipfli** [reply]
And finally also https://www.linkedin.com/feed/update/urn:li:activity:7019043864111980544 (did not know how to post on linkedin as maplibre on my phone...)

**Comment by wipfli** [comment]
Alan Wei Chen from [GrabMaps](https://www.grab.com/sg/business/maps/) is giving a talk about how Grab renders text labels in the Khmer language using FreeType and HarfBuzz in the next MapLibre Eastern Call. Feel free to join the presentation! 

https://maplibre.org/news/2023-01-12-grab-talk-khmer/

![image](https://user-images.githubusercontent.com/53421382/212111060-3ec9b082-4f6c-4d71-b806-232729c066b0.png)

**Reply by lseelenbinder** [reply]
LGTM

**Reply by nyurik** [reply]
Lgtm

**Reply by wipfli** [reply]
https://twitter.com/maplibre/status/1613572868251475968 and https://www.linkedin.com/feed/update/urn:li:activity:7019339120644628481
